app/posts/2021-09-01-equality-diversity.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Sharing equality and diversity data
+description: Sharing jobseeker equality and diversity data with schools.
+date: 2021-09-01
+related:
+  items:
+  - text: User research with medium to large MATs
+    href: https://docs.google.com/presentation/d/1jAb4tK4DOLEVkkFKowrEu5ibSYmBmpNizT3BR9s9QAU/edit#slide=id.ge6f7064364_0_347
+  - text: User research on the end-to-end application journey
+    href: https://docs.google.com/presentation/d/1kvZcFvz_qq3ckESeCjfBf7xaWdpE_HGc67pjACv9ipk/edit#slide=id.gb6d7927eb0_0_196
+  - text: Guidance from the Information Commissioner’s Office
+    href: https://ico.org.uk/media/1061/anonymisation-code.pdf
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+## Introduction
+
+When jobseekers apply for a job using the Teaching Vacancies application form, they are presented with optional diversity monitoring questions as part of the application process. When the application form is submitted, we ensure the diversity data is immediately separated from the application form, anonymised, and stored in a secure database. In user research, hiring staff from schools told us they would like to view this information to gauge how well their recruitment strategy is performing and ensure their workforce is diverse and inclusive.
+
+###  What we wanted to learn
```

Comment:
I get why you did it but the different heading sizes look a bit odd. Looking at other history entries, they are all ##